package.json

```diff
@@ -36,6 +36,7 @@
 		"class-validator": "^0.14.1",
 		"dotenv": "^16.4.5",
 		"echarts": "^5.5.0",
+		"express": "^4.19.2",
```

Comment:
I dont think you need to add this dependency. Pls try to do the statement `const response = context.getResponse<Response>();` without the type it should be automatically interfered. 
Add a large package just for a type seems to be an overhead to me.

If it is not possible *ignore this comment* and resolve it.